content/asset-tracking/index.textile

```diff
@@ -0,0 +1,163 @@
+- @nearWithSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints, with one or more subscribers.
+
+h2(#resolution-constraints). Resolution constraints
+
+Resolution constraint objects, @ResolutionConstraints@, are defined on a per-@Trackable@ basis, and are then used to calculate an appropriate @Resolution@ by the @ResolutionPolicy@.
+
+h3(#default-resolution-constraints). Default resolution constraints
+
+Ably provides default resolution constraints consisting of:
+
+- @DefaultResolutionSet@ := The set of resolutions which must be defined in order to populate the @DefaultResolutionConstraints@.
+- @DefaultProximity@ := This is the proximity to the destination at which resolution of updates is increased to improve user experience. Proximity can be spatial, temporal, or both. 
+- @batteryLevelThreshold@ := This is a level of battery on the device that runs Publishing SDK that is considered low. When the battery level falls below this, the resolution of updates is reduced.
+- @lowBatteryMultiplier@ := The multiplier to be applied to the interval when the battery level is below @batteryLevelThreshold@.
+
+The following example shows how the default resolution constraints are populated:
+
+```[android]
+// Prepare Resolution Constraints for an asset that will be used in the Resolution Policy
+val exampleConstraints = DefaultResolutionConstraints(
+    DefaultResolutionSet( // this constructor provides one Resolution for all states
+        Resolution(
+            accuracy = Accuracy.BALANCED,
+            desiredInterval = 1000L,
+            minimumDisplacement = 1.0
+        )
+    ),
+    proximityThreshold = DefaultProximity(spatial = 1.0),
+    batteryLevelThreshold = 10.0f,
+    lowBatteryMultiplier = 2.0f
+)
+```
+
+h2(#resolution-policy). Resolution policy
+
+Resolution policy defines the strategy by which the various @ResolutionRequests@ and preferences are translated by @Publisher@ instances into a target @Resolution@.
+
+As different use cases have different requirements for battery optimization, Ably Asset Tracking SDKs provide a flexible @ResolutionPolicy@ interface, that allows SDK users to provide their own implementation of resolution optimization logic. Ably also provides a "default resolution policy":#default-resolution-policy, which is suitable for common use cases.
+
+h3(#default-resolution-policy). Default resolution policy
+
+Ably Asset Tracking SDKs come with a @DefaultResolutionPolicy@ implementation provided out of the box, which is suitable for common use cases.
+
+Each @Trackable@ added to the @Publisher@ has a destination and set of @ResolutionConstraints@. Both @destination@ and @resolutionConstraints@ are optional for @Trackable@, but having them improves the resolution policy. Ably provides an optional set of "default resolution constraints":#default-resolution-constraints, @DefaultResolutionContraints@.
```

Comment:
"improves the resolution policy" -> "enables the resolution policy to reflect those constraints" maybe